### ISSUE TITLE
mkv didn't play

### ISSUE THREAD

**Issue by EikeBierman** [RIAEvangelist/electron-video-player#30]
i have download the mkv 20 MB for 1280x720 Resolution on this site:
http://www.sample-videos.com/index.php#sample-mkv-video

but they can't load it

**Comment by RIAEvangelist** [comment]
I assume you can play it after you give it a moment to load?

**Comment by EikeBierman** [comment]
i've try it against and wait ca 5 minutes... nothing happend

do you test it with the sample video ?

**Comment by profgrammer** [comment]
Yes, even I tried playing an MKV file, and it says error playing video
